### PR TITLE
Update markup, add styling for header and hero section, update class naming convention

### LINKED ISSUE #29: Build the mobile header

### LINKED ISSUE #30: Build the mobile hero section

### PR DESCRIPTION
**Pull request by gmagnenat**
ss_header

### CODE REVIEW

**Comment by gmagnenat** [comment]
I added the compiled css and html path to the .gitignore file. So it's not pushed to the repo at each update.

**Comment by KingLyrics** [comment]
Alright! I  will go through the code and see if anything needs to be corrected

File: src/sass/main.scss [hunk 1]
```diff
@@ -60,3 +92,58 @@ $ls-8: -0.45px;
...
+
+// hero images
+.hero-imgs {
+	background-color: $clr-neutral-800;
+	position: relative;
+	@include width-full(375px);
+
+	.app-img {
+		position: relative;
+		top: -20%;
```
Comment (KingLyrics) [inline src/sass/main.scss]: I thought this was going to be absolute as well? Since the .hero-imgs is already relative. Not a correction just an inquiry...

Comment (gmagnenat) [reply src/sass/main.scss]: this isn't in the last commit. it was a work in progress and some tests. it is removed after. we'll work on that after.

**Review by KingLyrics — COMMENTED** [review]
Alot of mixins and variables to go through lol, but it really helps simplifying the code. Other than that no issues from what i have reviewed. Any changes on the code can later on be tackled. So far, excellent work @gmagnenat !

File: src/sass/main.scss (hunk 1 above)
Comment (KingLyrics) [reply src/sass/main.scss]: Okay no problem at all

**Review by KingLyrics — APPROVED** [review]
All changes are great for now.

**Comment by gmagnenat** [comment]
close #30

**Pull request merged**